File: TextAnalysisAndroid/app/src/main/java/com/likhtman/TextAnalysis/Entity/TwoSenteces.java
package com.likhtman.TextAnalysis.Entity;

public class TwoSenteces {
    private String _firstSentence;
    private String _secondSentence;

    public TwoSenteces() {

    }

    public TwoSenteces(String tmpFirstSentence, String tmpSecondSentence) {
        setFirstSentence(tmpFirstSentence);
        setSecondSentence(tmpSecondSentence);
    }

    public String getFirstSentence() {
        return _firstSentence;
    }

    public void setFirstSentence(String _firstSentence) {
        this._firstSentence = _firstSentence;
    }

    public String getSecondSentence() {
        return _secondSentence;
    }

    public void setSecondSentence(String _secondSentence) {
        this._secondSentence = _secondSentence;
    }


}

File: TextAnalysisNetServer/Entity/TwoSenteces.cs
using System.Runtime.Serialization;

namespace TextAnalysis
{
	[DataContract]
	public class TwoSenteces
	{
		private string _firstSentence;
		private string _secondSentence;

		public TwoSenteces()
		{
		}

		public TwoSenteces(string tmpFirstSentence, string tmpSecondSentence)
		{
			firstSentence = tmpFirstSentence;
			secondSentence = tmpSecondSentence;
		}

		[DataMember]
		public string firstSentence
		{
			get { return _firstSentence; }
			set { _firstSentence = value; }
		}

		[DataMember]
		public string secondSentence
		{
			get { return _secondSentence; }
			set { _secondSentence = value; }
		}

		public override string ToString()
		{
			return
				firstSentence + " " + secondSentence;
		}
	}
}
